### DISCUSSION TITLE
If the input is multidimensional, how should I train the model?

### DISCUSSION THREAD

**Discussion by huangkr03** [OptimalScale/LMFlow#609] [Q&A]
For example, if I want it to take a long text and a set of keywords as input and make improvements to the long text as output, based on these keywords, how should I train the model? And which model should I choose?
I would greatly appreciate it if someone could give me some advice!

**Comment by shizhediao** [comment]
I suggest combining these inputs (i.e., a long text and a set of keywords) into a fluent sentence, which will be deemed as the input.

**Reply by huangkr03** [reply]
I apologize for my lack of understanding in this area, but I would like to inquire about the potential impact on the generation results of the model by doing so. 
And how should the combination process be approached? Are there any specific considerations to keep in mind during the training process?
Thank you for your reponse!
